### DISCUSSION TITLE
A Few More Bugs to Work Out...

### DISCUSSION THREAD

**Discussion by DominusExMachina** [bcholmes/StarTrek2d20#233] [General]
Well, tried the newest version of the creator (v1.240918), and, as the title of this thread suggests, found a few more bugs:

1.) When the Prototype service record is selected, the program then continues to the refits screen and will go no further until the refits (as per the Experimental Vehicle special rule) are selected: the refit screen, however, will not allow the user to do so.

2.) The descriptive text for the Sophisticated Astrometric Facilities starship talent is missing from the talent selection screen for the starship creator.

3.) The descriptive text for the Expanded Emergency Medical Facilities starship talent should read thusly:

"Whenever a character attempts a task to identify specific illnesses or injuries, or to determine the severity of a patient’s condition, they may diagnose a number of patients equal to the ship’s Medical rating. If the character also has the Triage talent, they may diagnose two additional patients by spending 1 Momentum (Repeatable).

During an extended task to process and treat many sick or injured patients, a character aboard the ship increases their Impact by 1."

4.) The descriptive text for the Improved Reaction Control System starship talent should read thusly:

"Whenever the ship attempts to move through difficult terrain, reduce the Momentum cost of the difficult terrain by 1, to a minimum of 0."

5.) When the "New 2e-style" is selected for a starship sheet, any text that doesn't fit on the single page is cut off entirely.

6.) The _Mogai_-class Romulan warbird (which the program still has listed as the _Valdore_-class) has a launch year of 2374: the program, however, has the launch date for the class listed as 2285.

**Comment by BaronVonStevie01** [comment]
Orions are no longer an option in character creation.

Starships, I believe, have a couple issues:

1. Starship weapons don't calculate correctly. The Damage bonus from the weapons rating isn't being added.
2. Systems are a point off on at least the Constitution Class. I tried a few classes in the creator and that was the only one.

**Comment by DominusExMachina** [comment]
Well, the new version (v1.241027) is out, and I found at least two issues with the starship creator:

1,) The _Norway_, _Intrepid_, _Sovereign_ and _Luna_ classes still retain the Emergency Medical Hologram talent in their stats after 2380, when the talent is rendered obsolete due to the fact that _**all**_ Starfleet vessels are equipped with an EMH as standard.

2.) When creating a 2e-style sheet for an _Odyssey_-class ship, the program won't even create the sheet: the other sheet styles still work fine, though.

As an aside, the 2e Saucer Separation special rule can easily (and has yet to be applied to) the 1e _Springfield_ and _Steamrunner_ classes, and the 2e Saucer Separation and Reconnect special rule to the _Ross_ and _Sutherland_ classes: additionally, a variation on the 2e Four-Nacelle Stability special rule could also be appled to the 1e _Prometheus_ class in the spaceframe's fully-assembled state.

**Reply by bcholmes** [reply]
Here are the 2e sheets I just generated from an Odyssey.
[USS_Hestia-starship.pdf](https://github.com/user-attachments/files/17542301/USS_Hestia-starship.pdf)
[USS_Hestia-starship(1).pdf](https://github.com/user-attachments/files/17542302/USS_Hestia-starship.1.pdf)

**Reply by DominusExMachina** [reply]
I saw the sheets you generated: the second sheet, which is the "New 2e" style, is fine, whereas with the sheet done using what I call the "Original 2e" style, the text for the special rules is cut off.

**Reply by bcholmes** [reply]
Sadly, the standard sheet just doesn't provide much room for special rules. There's not really anything I can do about that while maintaining the design of the original sheet.
